documentation/docs/14-configuration.md

```diff
@@ -91,6 +92,27 @@ export default config;
 
 Required when running `svelte-kit build` and determines how the output is converted for different platforms. See [Adapters](/docs/adapters).
 
+### alias
+
+An object containing zero or more aliases used to replace values in `import` statements, where each key is the alias and the value is a relative path to the alias value. These aliases are automatically passed to Vite and TypeScript.
```

Comment:
Initially I just removed the word 'relative' (since you could use an absolute path here and it should work fine) but that left us with 'the value is a path to the alias value' which is borderline meaningless 😆 I think it's easier to just let the code sample do the talking

```suggestion
An object containing zero or more aliases used to replace values in `import` statements. These aliases are automatically passed to Vite and TypeScript.
```